countries.json

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
I'm not sure why it was a module. :thinking: 
With the typing it's more easy to get the value by key (using `COUNTRY_CODE`) with a json than with an array of Namespaces.

Reply:
> I'm not sure why it was a module.

I'm also not sure 😉 
This is still from the original sucks library...

> thinking With the typing it's more easy to get the value by key (using `COUNTRY_CODE`) with a json than with an array of Namespaces.

I agree that it is a good idea to change it 👍🏻